### PR TITLE
Add libqt5serialport5-dev

### PR DESCRIPTION
**Pull request by carebare47**
Add support for the dev version of libqt5serialport5

### CODE REVIEW

**Review by ivanpauno — CHANGES_REQUESTED** [review]
Please, provides links to confirm the package names (e.g. https://packages.ubuntu.com/search?keywords=libqt5serialport5), and consider adding gentoo and arch (which together with debian/fedora/ubuntu are the recommended distros)

**Comment by mabelzhang** [comment]
Looks like
Debian (Jessie to Buster): https://packages.debian.org/bullseye/libqt5serialport5-dev
Ubuntu (Xenial to Focal): https://packages.ubuntu.com/focal/libqt5serialport5-dev
Fedora https://fedora.pkgs.org/32/fedora-x86_64/qt5-qtserialport-devel-5.13.2-2.fc32.i686.rpm.html

In addition, there is a Gentoo package https://packages.gentoo.org/packages/dev-qt/qtserialport
Please confirm and add Gentoo if that's the correct one.

File: rosdep/base.yaml [hunk 1]
```diff
@@ -3919,6 +3919,10 @@ libqt5-serialport:
   freebsd: [qt5-serialport]
   gentoo: ['dev-qt/qtserialport:5']
   ubuntu: [libqt5serialport5]
+libqt5-serialport-dev:
+  debian: [libqt5serialport5-dev]
+  fedora: [qt5-qtserialport-devel]
```
Comment (jacobperron) [inline rosdep/base.yaml:3924]:
```suggestion
  fedora: [qt5-qtserialport-devel]
  gentoo: [dev-qt/qtserialport]
```

**Comment by clalancette** [comment]
No response in over a month, so I'm going to close this.  Please feel free to respond to the review comments and reopen.  Thanks for the contribution.

**Pull request closed without merging**